test/test_api.py

```diff
@@ -12,63 +12,65 @@
 
 current_path = Path(os.path.dirname(os.path.realpath(__file__)))
 
-gridfile_ne30 = current_path / "meshfiles" / "ugrid" / "outCSne30" / "outCSne30.ug"
-dsfile_var2_ne30 = current_path / "meshfiles" / "ugrid" / "outCSne30" / "outCSne30_var2.nc"
-dsfiles_mf_ne30 = str(
-    current_path) + "/meshfiles/ugrid/outCSne30/outCSne30_*.nc"
 
-gridfile_geoflow = current_path / "meshfiles" / "ugrid" / "geoflow-small" / "grid.nc"
+class TestAPI(TestCase):
 
+    geoflow_data_path = current_path / "meshfiles" / "ugrid" / "geoflow-small"
+    gridfile_geoflow = current_path / "meshfiles" / "ugrid" / "geoflow-small" / "grid.nc"
+    geoflow_data_v1 = geoflow_data_path / "v1.nc"
+    geoflow_data_v2 = geoflow_data_path / "v2.nc"
+    geoflow_data_v3 = geoflow_data_path / "v3.nc"
 
-class TestAPI(TestCase):
+    gridfile_ne30 = current_path / "meshfiles" / "ugrid" / "outCSne30" / "outCSne30.ug"
+    dsfile_var2_ne30 = current_path / "meshfiles" / "ugrid" / "outCSne30" / "outCSne30_var2.nc"
+    dsfiles_mf_ne30 = str(
+        current_path) + "/meshfiles/ugrid/outCSne30/outCSne30_*.nc"
 
     def test_open_geoflow_dataset(self):
         """Loads a single dataset with its grid topology file using uxarray's
         open_dataset call."""
 
-        # Base data path
-        base_path = "test/meshfiles/ugrid/geoflow-small/"
-
-        # Path to Grid file
-        grid_path = base_path + "grid.nc"
-
         # Paths to Data Variable files
-        var_names = ['v1.nc', 'v2.nc', 'v3.nc']
-        data_paths = [base_path + name for name in var_names]
+        data_paths = [
+            self.geoflow_data_v1, self.geoflow_data_v2, self.geoflow_data_v3
+        ]
 
-        uxds_v1 = ux.open_dataset(grid_path, data_paths[0])
+        uxds_v1 = ux.open_dataset(self.gridfile_geoflow, data_paths[0])
 
     #     TODO: Add asserts
```

Comment:
Do we plan to add the assert through another PR?

Reply:
I personally didn't make this test case, but I would assume so.